frontend/src/components/rankingTeamList.js

```diff
@@ -31,40 +95,63 @@ class RankingTeamList extends TeamList {
           buttonContent = <i className="fa fa-check"></i>;
         }
         return (
-          <tr key={team.id} onClick={() => this.redirectToTeamPage(team.id)}>
+          <tr
+            key={team.id}
+            onClick={() => this.redirectToTeamPage(team.id)}
+            className="page-item"
+          >
             {<td>{Math.round(team.profile.rating)}</td>}
             <td>{team.name}</td>
             <td>{team.members.map((member) => member.username).join(", ")}</td>
-            {<td>{team.profile.quote}</td>}
-            <td>
-              {this.props.episode_info.eligibility_criteria.map((criterion) => {
-                const eligible = team.profile.eligible_for.includes(
-                  criterion.id
-                );
-                return (
-                  <span key={criterion.id}>
-                    {eligible ? criterion.icon : ""}
-                  </span>
-                );
-              })}
-            </td>
-            {/* <td>{team.auto_accept_unranked ? "Yes" : "No"}</td> */}
-            {/* {props.canRequest && (
+            {!this.props.canRequest && <td>{team.profile.quote}</td>}
+            {!this.props.canRequest && (
               <td>
-                <button
-                  className="btn btn-xs"
-                  onClick={() => this.onTeamRequest(team.id)}
-                >
-                  {buttonContent}
-                </button>{" "}
+                {this.props.episode_info.eligibility_criteria.map(
+                  (criterion) => {
+                    const eligible = team.profile.eligible_for.includes(
+                      criterion.id
+                    );
+                    return (
+                      <span key={criterion.id}>
+                        {eligible ? criterion.icon : ""}
+                      </span>
+                    );
+                  }
+                )}
               </td>
-            )} */}
+            )}
+            {this.props.canRequest && (
+              <td>{team.profile.auto_accept_ranked ? "Yes" : "No"}</td>
+            )}
+            {this.props.canRequest && (
+              <td>{team.profile.auto_accept_unranked ? "Yes" : "No"}</td>
+            )}
+            {this.props.canRequest &&
+              this.props.team &&
+              team.id !== this.props.team.id && (
+                <td>
+                  <button
+                    className="btn btn-xs"
+                    onClick={(event) => {
+                      event.stopPropagation();
+                      this.openRequestForm(team);
+                    }}
+                  >
+                    {buttonContent}
+                  </button>{" "}
+                </td>
+              )}
```

Comment:
Do you think it'd be possible to move the `<td>...</td>` tags outside the `canRequest` conditional? I imagine you'd want a blank cell to show up if you can't request, rather than no cell (like it is now, see screenshot)

<img width="508" alt="image" src="https://user-images.githubusercontent.com/16425187/210023886-d81c2975-01a5-4ec9-8727-69f948471630.png">

Reply:
`canRequest` is about whether to show the whole column in the first place, so that's ok, but we should keep the td tag when conditionally rendering dependent on if it's their own team, so have fixed that